SupportClient/SupportClient/Controllers/ViewController.swift

```diff
@@ -0,0 +1,198 @@
+        newConversationButton.titleLabel?.font = UIFont(name: Constants.robotoMedium, size: 17)
+        newConversationButton.layer.cornerRadius = 22
+        view.addSubview(newConversationButton)
+        
+    }
+    
+    func setUpFeedbackTableView() {
+        
+        feedbackTableView = UITableView(frame: .zero)
+        feedbackTableView.translatesAutoresizingMaskIntoConstraints = false
+        feedbackTableView.delegate = self
+        feedbackTableView.dataSource = self
+        feedbackTableView.register(FeedbackTableViewCell.self, forCellReuseIdentifier: reuseID)
+        view.addSubview(feedbackTableView)
+                    
+    }
+    
+    func setUpConstraints() {
+                
+        if isFeedback {
+            
+            feedbackTableView.snp.makeConstraints{ (make) -> Void in
+                make.leading.trailing.top.bottom.equalToSuperview()
+            }
+            
+        } else {
+            
+            feedbackTableView.isHidden = true
+            
+            newConversationButton.snp.makeConstraints{ (make) -> Void in
+                make.centerX.equalToSuperview()
+                make.bottom.equalToSuperview().inset(50)
+                make.width.equalTo(view.layer.bounds.width / 2)
+                make.height.equalTo(45)
+            }
+            
+            feedbackLabel.snp.makeConstraints{ (make) -> Void in
+                make.centerX.equalToSuperview()
+                make.bottom.equalToSuperview().inset(view.layer.bounds.height/2)
+            }
+            
+        }
+        
+    }
+    
+    @objc func handleNavigationBarRightTap() {
+        
+        if countEditTaps % 2 == 0 {
+            feedbackTableView.setEditing(true, animated: true)
+        } else {
+            feedbackTableView.setEditing(false, animated: true)
+        }
+        countEditTaps += 1
+
+    }
+    
+}
+
+
+extension ViewController: UITableViewDelegate {
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return cellHeight
+    }
+    
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let cell = feedbackTableView.cellForRow(at: indexPath) as! FeedbackTableViewCell
+        let feedback = feedbackData[indexPath.row]
+        feedback.hasUnread.toggle() // TODO: remove this once hooked up to database
+        cell.toggleRead(for: feedback) // TODO: remove this once hooked up to database
+    }
+    
+    func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
+        if editingStyle == .delete {
+            feedbackData.remove(at: indexPath.row)
+            feedbackTableView.deleteRows(at: [indexPath], with: .fade)
+        }
+        // if all conversations removed, go back to default page
+        if feedbackData.count == 0 {
+            isFeedback = false
+            setUpConstraints()
```

Comment:
if you're calling `setupConstraints` here, then that means you need to do `remakeConstraints` instead of `makeConstraints` in the function for all your views so that you don't end up making the same constraints again. `remakeConstraints` will remove all current constraints on the view and make them again